docs/src/release_notes/v1.15.md

```diff
@@ -6,6 +6,42 @@ For a complete list of changes, please refer to the
 [commits](https://github.com/cloudnative-pg/cloudnative-pg/commits/release-1.15)
 on the release branch in GitHub.
 
+
+##  Version 1.15.3
+
+**Release date:** Aug 11, 2022
+
+Enhancements:
+
+- Enable the configuration of the `huge_pages` option for PostgreSQL (#456)
+- Enhance log during promotion and demotion, after a failover or a switchover,
+  by printing the time elapsed between the request of promotion and the actual
+  availability for writes (#371)
+- Add the `instanceName` and `clusterName` labels on jobs, pods, and PVCs to
+  improve interaction with these resources (#534)
+- Add instructions on how to create PostGIS clusters
+
+Security:
+
+- Explicitly assign `securityContext` to the `Pooler` deployment (#485)
+- Add read timeout values to the internal web servers to prevent Slowloris DDoS (#437)
+
+Fixes:
+
+- Use the correct delays for restarts (`stopDelay`) and for switchover
+  (`switchoverDelay`), as they were erroneously swapped before. This is an
+  important fix, as it might block indefinitely restarts if `switchoverDelay` is
+  not set and uses the default value of 40000000 seconds (#531)
+- Prevent the metrics collector from causing panic when the query returns an
+  error (#396)
+- Removing an unsafe debug message that was referencing an unchecked pointer,
+  leading in some cases to segmentation faults regardless of the log level (#491)
+- Prevent panic when fencing in case the cluster had no annotation (#512)
+- Avoid updating the CRD if a TLS certificate is not changed (#501)
+- Handle conflicts while injecting a certificate in the CRD (#547)
+- Backup and recovery:
+    - Correctly pass object store credentials in Google Cloud (#454)
+
```

Comment:
I think we can probably add a section like "thanks to contributors" or mention people that contribute with code or ideas for this release